src/main/java/org/opentripplanner/raptor/path/PathBuilder.java

```diff
@@ -58,10 +58,12 @@
   // paths with the same path tail.
   private PathBuilderLeg<T> head = null;
   private PathBuilderLeg<T> tail = null;
+  protected final int iterationDepartureTime;
```

Comment:
The name `iterationDepartureTime` is probably a bit off, but I do not have a better suggestion. At least the name indicate where this time is coming from, but `earliestDepartureTime` would probably be "as correct". So maybe `fallbackDepartureTime` is better. I vote for `iterationDepartureTime` even if it is not 100% correct.